google/resource_compute_firewall.go

```diff
@@ -254,6 +258,12 @@ func resourceComputeFirewallCreate(d *schema.ResourceData, meta interface{}) err
 	} else if v, ok := d.GetOkExists("disabled"); ok || !reflect.DeepEqual(v, disabledProp) {
 		obj["disabled"] = disabledProp
 	}
+	enableLoggingProp, err := expandComputeFirewallEnableLogging(d.Get("enable_logging"), d, config)
+	if err != nil {
+		return err
+	} else if v, ok := d.GetOkExists("enable_logging"); ok || !reflect.DeepEqual(v, enableLoggingProp) {
+		obj["enableLogging"] = enableLoggingProp
```

Comment:
do you need to cast this to a simple Type or is it ok as an interface still?

Reply:
object is a map[string]interface{} so I think this would just end up getting cast back to interface anyways